app/assets/javascripts/charts.js

```diff
@@ -259,7 +260,12 @@ function get_pie_chart(div, url) {
       $.getJSON(url, function(data) {
         var target = $("#"+div+"-body");
         target.empty();
+        var hostsCount = 0;
+        $.each(data.values,function() {
+          hostsCount += this.data;
+        });
         expanded_pie(target, data.values);
+        $('.modal-title').empty().append('<b>' + data.name + ' </b>' +  __('Distribution chart') + '<small> ('+ Jed.sprintf(n__("%s host", "%s hosts", hostsCount), hostsCount) +')</small>');
```

Comment:
I think this looks a bit nicer as `Fact distribution chart - fact_name
:+1:
![](http://i.imgur.com/v60oYKl.png)
:+1:
![](http://i.imgur.com/OCpuZh8.png)

cc @domcleal @ohadlevy your thoughts?